auditbeat/module/file_integrity/metricset.go

```diff
@@ -177,6 +179,32 @@ func (ms *MetricSet) init(reporter mb.PushReporterV2) bool {
 	return true
 }
 
+// Determine which - if any - paths  have been newly added to the config
```

Comment:
The comments are great. I just want to encourage you to use the godoc format (even for private methods) which is to start the sentence with the name of the method (e.g. `findNewPaths finds the new paths...`).

Reply:
Thanks for the pointer, I fixed it here and will try to remember going forward.